public/Day 07/index.html

```
@@ -76,18 +40,59 @@ <h1>💸 Expense Tracker</h1>
+        <div class="empty-state">No expenses yet. Add your first expense above!</div>
+      </div>
     </div>
 
-    <h3 class="list-title">Your Expenses</h3>
-    <div id="expenses-container" class="expenses-container">
-      <p class="empty">No expenses added yet.</p>
 
+    <div class="container">
+      <h1>💸 Expense Tracker</h1>
+
+      <form id="expense-form">
```

HTMLHint (1.9.1)

[error] The id value [ expense-form ] must be unique.

(id-unique)